### PR TITLE
Mod4 Pull Wars

### PR DESCRIPTION
**Pull request by falonofthetower**
There is no try, there is only pull or do not pull

### CODE REVIEW

File: app/views/videos/show.html.haml [hunk 1]
```diff
@@ -19,7 +19,7 @@
           .form-group
             .row
               .col-sm-3
-                = f.select :rating, options_for_select([5,4,3,2,1].map {|number| [pluralize(number, "Star")]}), { label: "Rate this video" }
+                = f.select :rating, options_for_select((1..5).map {|number| [pluralize(number, "Star"), number]}), { label: "Rate this video" }
```
Comment (brandoncc) [inline app/views/videos/show.html.haml]: How come you are not using your helper?

File: spec/controllers/queue_items_controller_spec.rb [hunk 2]
```diff
@@ -102,5 +102,87 @@
...
+
+      it "sets the flash danger message" do
+        post :update_queue, queue_items: [{id: light_saber.id, position: 3.4}, {id: r2d2.id, position: 1}]
+        expect(flash[:danger]).to be_present
+      end
+
+      it "does not change the queue items" do
+        post :update_queue, queue_items: [{id: light_saber.id, position: 3}, {id: r2d2.id, position: 2.1}]
+        expect(light_saber.reload.position).to eq(1)
+        expect(response).to redirect_to my_queue_path
```
Comment (brandoncc) [inline spec/controllers/queue_items_controller_spec.rb]: No need for this second expectation. I'm guessing you copy and pasted?

File: spec/models/queue_item_spec.rb [hunk 3]
```diff
@@ -37,6 +38,34 @@
...
+        queue_item.rating = nil
+        expect(Review.first.rating).to be_nil
+      end
+
+      it "creates a review with the rating if the review is not present" do
+        a_new_hope = Fabricate(:video)
+        luke = Fabricate(:user)
+        queue_item = Fabricate(:queue_item, user: luke, video: a_new_hope)
+        queue_item.rating = 3
+        expect(Review.first.rating).to eq(3)
```
Comment (brandoncc) [inline spec/models/queue_item_spec.rb:65]: This spec doesn't really do what is described. I would change the description to something like `it "assigns the correct rating to the new review"`, then create another spec with the same setup and description as this one, and use this expectation:

``` ruby
expect { queue_item.rating = 3 }
  .to change { Review.count }.from(0).to(1)
```

Alternatively, in this one case I would be okay with replacing line 64 with the line above and having two expectations in this spec.

Comment (falonofthetower) [inline spec/models/queue_item_spec.rb:65]: Will  need to revisit this

**Pull request merged**